.github/workflows/arbitrator-ci.yml

```diff
@@ -5,10 +5,7 @@ on:
   pull_request:
     paths:
       - 'arbitrator/**'
-      - 'contracts/src/osp/**'
-      - 'contracts/src/mock/**'
-      - 'contracts/test/**'
-      - 'contracts/hardhat.config.ts'
+      - 'contracts'
```

Comment:
I'd like to keep our previous conditions for arbitrator CI if only relevant parts of the contracts were changed, but I don't think that's easily possible with it being a submodule now. In theory maybe the workflow could exit early with success if it detects nothing's changed? But I'm also not sure how big of an issue this will be. We could also not run it on contracts changes, but we've had contracts changes that have broken it before so I think it's safer to keep it running.